src/sql/workbench/services/serverGroup/browser/serverGroupDialog.ts

```diff
@@ -196,7 +196,7 @@ export class ServerGroupDialog extends Modal {
 				name: 'server-group-color',
 				color: color
 			});
-
+			this._register(colorBox);
```

Comment:
nit/suggestion: You can do this inline with the creation as well (I prefer that so that it's easy to ensure that every disposable created is also registered at a glance)